lib/authenticate.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const opn = require('opn')
+const randtoken = require('rand-token')
+const websocket = require('websocket-stream')
+const split2 = require('split2')
+const { ReadableStreamBuffer } = require('stream-buffers')
+const { URL } = require('url')
+const pump = require('pump')
+
+/**
+ * Get the JWT token from the Upload server specified by the URL.
+ * - create a random tokens
+ * - open a websocket on the server
+ * - push the token through the websocket
+ * - Open the browser on `http://localhost:3000/?token=${cliToken}`
+ * - Get the JWT from the web websocket
+ */
+const authenticate = async url =>
```

Comment:
please remove `async` from here, there is no need.

Reply:
indeed